benchmark-compare.js

```diff
@@ -1,34 +1,69 @@
 #!/usr/bin/env node
 'use strict'
 
+const os = require('os')
+const commander = require('commander')
 const inquirer = require('inquirer')
-const chalk = require('chalk')
 const Table = require('cli-table')
+const chalk = require('chalk')
 const { join } = require('path')
 const { readdirSync, readFileSync, writeFileSync } = require('fs')
-const { compare } = require('./lib/autocannon')
 const { info } = require('./lib/packages')
-const commander = require('commander')
+const { compare } = require('./lib/autocannon')
+
+const resultsPath = join(process.cwd(), 'results')
 
 commander
-  .option('-t, --table', 'table')
-  .option('-p, --percentage', 'percentage')
-  .option('-c --commandlineMdTable', 'Print a table for use in MarkDown')
+  .option('-t, --table', 'print table')
+  .option('-m --markdown', 'format table for markdown')
```

Comment:
How this action should work? I've tested locally but seems not to work.

```sh
rafaelgss@xxx:~/repos/os/benchmarks$ node ./benchmark compare -m
? What's your first pick? express
? What's your second one? fastify

 Both are awesome but fastify is 364% faster than express
 • fastify request average is 29113.6
 • express request average is 6274.43
```

Reply:
`benchmark compare -t [-m]` renders comparison table while the default behavior is interactive mode (as it was)

In github action, `compare -u` is used which updates `README.md` (logic moved from GitHub action steps)